cloud-logs-streamer/Jenkinsfile_Platform

```diff
@@ -2,14 +2,18 @@
 import groovy.json.JsonOutput
 import groovy.transform.Field
 
+// To be replaced as @Field def repo_credential_id = "value" for repo_credential_id, repo_base and repo_core
+@Field def repo_credential_id
+@Field def repo_base
+@Field def repo_core
 @Field def configModule
 @Field def configLoader
 @Field def jenkins_credential_id
 @Field def region
 
 node {
-
-  loadConfigModule("http://${env.REPO_BASE}/scm/${env.REPO_CORE}/jazz-build-module.git")
+  jazzBuildModuleURL = "http://" + repo_base + "/scm/" + repo_core + "/jazz-build-module.git"
```

Comment:
use convention of single string with variables => "http://${repo_base}/scm/${repo_core}/jazz-build-module.git"

Reply:
make this change every place that this PR is adding/editing.